Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -125,19 +135,23 @@ open class NavigationMapView: UIView {
      
      The `UserCourseView`'s `UserCourseView.update(location:pitch:direction:animated:)` method is frequently called to ensure that its visual appearance matches the map’s camera.
      */
-    public var userCourseView: UserCourseView = UserPuckCourseView(frame: CGRect(origin: .zero, size: 75.0)) {
+    var userCourseView: UserCourseView? {
         didSet {
-            oldValue.removeFromSuperview()
-            installUserCourseView()
+            oldValue?.removeFromSuperview()
+            
+            if let userCourseView = userCourseView {
+                userCourseView.isHidden = false
+                mapView.addSubview(userCourseView)
+            }
         }
     }
     
     /**
      Specifies how the map displays the user’s current location, including the appearance and underlying implementation.
      
-     By default, this property is set to `UserLocationStyle.courseView`, the bearing source is location course.
+     By default, this property is set to `UserLocationStyle.puck2D(configuration:)`, the bearing source is location course.
      */
-    public var userLocationStyle: UserLocationStyle = .courseView(UserPuckCourseView(frame: CGRect(origin: .zero, size: 75.0))) {
+    public var userLocationStyle: UserLocationStyle? = .puck2D(configuration: nil) {
```

Comment:
`NavigationMapView.userLocationStyle` is now optional and in case of `nil` will not show current location indicator.